docs/user-guide-for-quickstart.md

```diff
@@ -0,0 +1,133 @@
+top left to open the details view of the navigation menu.
+
+<div class="my-carousel docs-carousel">
+    <img src="/img/quickstart/user-guide/01_home.png" alt="Home"/>
+</div>
+
+
+## Pipelines
+The _Pipelines_ view provides an overview of all existing pipelines.
+
+A pipeline in Apache StreamPipes describes the transformation process from a data stream to a data sink. Typically, a pipeline consists of at least one data stream (or data set), zero or more data processors and at least one data sink.
+Existing pipelines can be managed using this pipeline view.For instance, users can start and stop pipelines or delete them when they are not longer needed.
+In the pipeline page, we could see there are 5 pipelines, click 'START ALL PIPELINES' button, we could start all the predefined pipelines.
+
+<div class="my-carousel docs-carousel">
+    <img src="/img/quickstart/user-guide/02_pipelines.png" alt="Pipeline Overview"/>
+</div>
+
+
+ Using the 'Water level trend analysis' pipeline as an example. By clicking the 'Show pipeline' button, we could see the details of the pipeline as the following figure.
+ 
+
+The pipeline contains 3 major parts:
+- **Data source**: 
+  - Machine Data Simulator (Water Level).
+- **Data Processors**: 
+  - Trend Detector: to analyse the water level increase speed within specified time, when Trend Detector find the speed is reaching the limit, it will trigger the notification. 
+  - Welford Change Detection: to calculate the variance of the mean and change of the water level over a specified time 
+  - Boolean Counter: to count each 'overflow' status change from 'false' to 'true'. 
+  - Numerical Filter: when 'overflow' status change from 'false' to 'true' reaches 5 times, it will trigger Notification.
+- **Data Sinks**: 
+  - Notification: to push a notification when the measured parameter condition reaches the set condition of the trigger, which are sent to Notification interface. 
+  - Data Lake: to store the events in the internal data lake, offering data for dashboard and data explorer.
+
+<div class="my-carousel docs-carousel"> 
+<img src="/img/quickstart/user-guide/07_water_level_trend_pipeline.png" alt="Pipeline Detail"/>
+</div>
+
+## Connect
+The _Connect_ view provides an overview of all existing adapters to let StreamPipes connect data sources.
+
+With StreamPipes Connect it is possible to connect new data sources in StreamPipes with just a few clicks.
+It is also possible to connect specific data sources or connect generic sources like message brokers or databases.
+If the event schema of the data source is unknown, the system tries to infer the schema by extracting some sample data and analysing it.
+
+In the Quickstart Mode, we use the Machine Data Simulator and Data stream to simulate the factory machine condition data and environmental data, just click 'START ALL ADAPTERS' button, we could start all existing adapters to offer data source for pipelines.
```

Comment:
```suggestion
In the Quickstart Mode, we use the Machine Data Simulator and Data stream to simulate the factory machine condition data and environmental data, just click the `START ALL ADAPTERS` button. This starts all existing adapters to offer data sources for pipelines.
```

Reply:
Hi, @bossenti ! have changed the description as suggestion.